BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/SelfDiagnostics/SelfDiagnosticsConfigRefresher.cs

```diff
@@ -0,0 +1,120 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing.SelfDiagnostics
```

Comment:
`SelfDiagnosticsInitializer` is in namespace `Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing`, this class is in `Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing.SelfDiagnostics` namespace. Should both `SelfDiagnosticsInitializer` and `SelfDiagnosticsConfigRefresher` get as a part of same namespace?